email-templates/task3.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style media="screen">
+    .kit{
+        display: inline-block;
+        padding: 10px 20px;
+        background-color: #2838e7;
+        color: #ffffff;
+        text-decoration: none;
+        border-radius: 5px;
+     }
+     h1 {
+    color: #ead9dd;
+    font-size: 28px; 
+    font-weight: bold; 
+    text-transform: uppercase; 
+    margin-bottom: 10px;
+  }
+     header {text-align: center;
+      padding-bottom: 20px;
+      background-color: rgb(18, 18, 18);
+     }
+      .body{text-align: center;
+      padding-bottom: 20px;
+      background-color: green;
+     }
+      .logo{text-align: center;
+      padding-bottom: 20px;
+     }
+     .main-content {
+        text-align: center;
+      padding: 20px;
+      text-align: center;
+    
+    }
+    .button{ text-align: center;
+      padding: 20px;
+      background-color: red;
+
+    }
+
+
+
+        </style>
+      </head>
+<body>
+    
+    <div class="container">
+        <header>
+            <h1>CRISTIANO RONALDO</h1>
+          </header>
+        
+     <div class="body">
+        <i>
```

Comment:
The order tag should be descending like `div > p > i/b/u`

Reply:
re arranged in the order